src/EPR.Accreditation.Common.Data/Migrations/20240510113820_Country-Code.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EPR.Accreditation.API.Common.Data.Migrations
+{
+    public partial class CountryCode : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "CountryCode",
+                table: "Accreditation",
+                type: "nvarchar(max)",
```

Comment:
Shouldn't be max. How many characters is country code going to be?

Reply:
Done.